.github/workflows/mirror-dev.yml

```diff
@@ -0,0 +1,34 @@
+name: Mirror dev to GitLab
+
+on:
+  push:
+    branches:
+      - dev
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - dev
+
+jobs:
+  mirror:
+    if: github.repository == 'accessibility-exchange/platform'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Git
+        run: |
+          git config --global user.name "${{ vars.GIT_USER }}"
+          git config --global user.email "${{ vars.GIT_EMAIL }}"
+          git remote add gitlab ${{ vars.GITLAB_URL }}
```

Comment:
Are these vars supposed to be secrets?

Reply:
No. I intended that they would be variables and not secrets since they don't need to be masked for security.

Reply:
Where do the vars get sources from?

Reply:
Same area as secrets but different tab.
![Screenshot_20230531_181754](https://github.com/accessibility-exchange/platform/assets/341606/758dc671-deb9-4915-a963-fbd62dc7ffc6)

Reply:
I didn't see these in the repo so I've added them. However, please update them as needed.